plugins/BEdita/Core/src/Model/Behavior/UserModifiedBehavior.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2016 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+namespace BEdita\CORE\Model\Behavior;
+
+use BEdita\Core\Utility\LoggedUser;
+use Cake\Datasource\EntityInterface;
+use Cake\Event\Event;
+use Cake\ORM\Behavior;
+
+/**
+ * UserModified behavior
+ *
+ * Set current logged user id in `created_by` field (on new entity creation)
+ * and in `modified_by` (on entity update, always) field.
+ *
+ * @since 4.0.0
+ */
+class UserModifiedBehavior extends Behavior
+{
+
+    /**
+     * Default config
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
```

Comment:
I would have probably preferred an approach more similar to that that can be seen in `TimestampBehavior`, as it provides more flexibility, but nevertheless this could be a future refactor of this behavior if and when we will actually need a more configurable and flexible class.